docs/modular-resources/advanced/create-subtype.md

```diff
@@ -4,76 +4,83 @@ linkTitle: "New API Subtype"
 weight: 30
 type: "docs"
 tags: ["rdk", "extending viam", "modular resources", "API"]
-description: "Define a new resource subtype using an existing resource API."
+description: "Define a new resource subtype and API for a resource that does not fit into existing component or service subtypes."
 no_list: true
 aliases:
   - "/extend/modular-resources/create/create-subtype/"
 ---
 
-{{% alert title="Tip" color="tip" %}}
+You can define a new resource _{{< glossary_tooltip term_id="subtype" text="subtype" >}}_ (and an API for that subtype) if:
```

Comment:
```suggestion
You can define a new resource _{{< glossary_tooltip term_id="subtype" text="subtype" >}}_ API if:
```
[opt] I realize these are not new words but maybe this would be a better way to put it that is less redundant? Because a new subtype _is_ a new API?